src/App.jsx

```diff
@@ -14,70 +16,65 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectProduct, setSelectProduct] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectProduct ? `${selectProduct} is selected` : `No goods selected`}
+        {selectProduct && (
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => setSelectProduct('')}
+          />
+        )}
```

Comment:
no need to use two conditions here. Please use first with react fragment
```suggestion
        {selectProduct 
          ? (
              <>
                 `${selectProduct} is selected` 
                  <button
                    data-cy="ClearButton"
                    type="button"
                    className="delete ml-3"
                    onClick={() => setSelectProduct('')}
                   />
           :  `No goods selected`}
        
```